src/databricks/labs/ucx/cli.py

```diff
@@ -103,6 +103,14 @@ def sync_workspace_info(a: AccountClient):
     ctx.account_workspaces.sync_workspace_info()
 
 
+@ucx.command(is_account=True)
+def upload(path: Path, w: WorkspaceClient, run_as_collection: bool = False, a: AccountClient | None = None):
+    """Upload a file to the (collection of) workspace(s)"""
+    logger.warning("The schema of CSV files is NOT validated, ensure it is correct")
```

Comment:
What do we think about this warning? Do we want to extend the logic, thus complicating the logic, to verify which files could be uploaded in a ucx installation and what there schemas are? Or is this fine(ish)